kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java

```diff
@@ -75,13 +73,12 @@ public void createConnector() {
                 .waitUntilScreenReady()
                 .setConnectorConfig(
                         SINK_CONNECTOR,
-                        FileUtils.getResourceAsString("config_for_create_connector.json"));
+                        Objects.requireNonNull(FileUtils.getResourceAsString("config_for_create_connector.json")));
```

Comment:
no need to place here Objects.requireNonNull because getResourceAsString() doesn't have return null option now

Reply:
Reverted them